network/receiver.go

```diff
@@ -51,10 +52,13 @@ func handleReceiving(log zerolog.Logger, wg *sync.WaitGroup, cfg *Config, peers
 			err = peers.Drop(address)
 			if err != nil {
 				log.Error().Err(err).Msg("could not drop peer")
+			} else {
```

Comment:
Please don't use an else statement. Refactor to keep the default case on the lowest level of indentation possible.